### PR TITLE
array-of-x Shorthand

### PR DESCRIPTION
**Pull request by erfanium**

### CODE REVIEW

File: test/validator.spec.js [hunk 1]
```diff
@@ -242,6 +242,11 @@ describe("Test getRuleFromSchema method", () => {
 			expect(res.schema).toEqual({ type: "string", empty: false, alpha: false, trim: true, some: "1234kg" });
 		});
 
+		it("should convert arrayOf syntax", () => {
+			const res = v.getRuleFromSchema("string[]");
+			expect(res.schema).toEqual({ type: "array", items: "string" });
+		});
```
Comment (icebob) [inline test/validator.spec.js:248]: And it can work like `"string[]|optional|min:1"`?

Comment (erfanium) [reply test/validator.spec.js:248]: My plan was not to support this for now, but okay, I'll test it

Comment (icebob) [reply test/validator.spec.js:251]: As I saw the code, I think it can work.

**Review by icebob — APPROVED** [review]
Awesome!

**Pull request merged**